_projects/SP19-101010.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: 101010
+image: https://drive.google.com/file/d/1ViBTIdTskJJas1hImA3Ytet1czVtIOTw/view?usp=sharing
```

Comment:
Please move the image to `/img/projects/` and link it as such here, it needs a direct link for embed and that is something Google Drive does not allow.

Reply:
Done.